haystack/nodes/summarizer/transformers.py

```diff
@@ -206,13 +194,13 @@ def predict_batch(
 
         :param documents: Single list of related documents or list of lists of related documents
                           (e.g. coming from a retriever) that the answer shall be conditioned on.
-        :param generate_single_summary: Whether to generate a single summary for each provided document list or
-                                        one summary per document.
-                                        If set to "True", all docs of a document list will be joined to a single string
-                                        that will then be summarized.
-                                        Important: The summary will depend on the order of the supplied documents!
+        :param generate_single_summary: This parameter is deprecated and will be removed in Haystack 1.12
```

Comment:
Let's add a mention to the fact that the DocumentMerger is taking over this task: 

```suggestion
        :param generate_single_summary: This parameter is deprecated and will be removed in Haystack 1.12. To obtain single summaries from multiple documents, consider using the [DocumentMerger](https://docs.haystack.deepset.ai/docs/document_merger)
```